engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/SchemaWithMetadata.scala

```diff
@@ -0,0 +1,10 @@
+package pl.touk.nussknacker.engine.avro.schemaregistry
+
+import org.apache.avro.Schema
+
+/**
+ * This class holds information that are stored next to schema in registry.
+ * It is lightened version of Confluent's SchemaMetadata. We don't want to use their class, because our SchemaRegistryClient
+ * is not coupled with concrete schema registry implementation.
+ */
+case class SchemaWithMetadata(schema: Schema, id: Int)
```

Comment:
- If we have `RuntimeSchemaData`then maybe better name will be just `SchemaData`?
- Hmm, I think about add there version: Option[Int]?

Reply:
Ad 1. I want to has similar naming as Confluent does. IMO it is no need to make it looks similar like  `RuntimeSchemaData` because there are for other purposes.
Ad 2. For now YAGNI, there is many other fields that we can have here, but IMO we shouldn't put all because we don't know how looks like other schema registry implementations. Take a look on full confluent's `SchemaMetada`:
```
    private int id;
    private int version;
    private String schemaType;
    private String schema;
    private List<SchemaReference> references;
```